### PR TITLE
Display info in tables according to data from

### PR DESCRIPTION
**Pull request by mariamera**
API

Closes [Ticket](https://trello.com)

## Testing

Steps to manually verify the change:
1. _Setup_: open `paradise-papers-django/search_frontend/index.html`
2. _Exercise_: click search. 
3. _Verify_: check if information through tabs are dynamically. 
PS: only the information, not the header text 👍 
ex : 
image

image

image

### CODE REVIEW

File: search_frontend/js/search.js [hunk 1]
```diff
@@ -30,20 +31,50 @@
...
+          if (key == this._node_type ) { 
+            for (var x in mockup_data.filterData[key]) 
+              list.push(x)   
+            }
+              this.structureData.push(list)  
+          }
+    } 
+
+    fillInformation(row) {
+      var list = []
```
Comment (junior1q94) [inline search_frontend/js/search.js]: Always use `let` instead of `var` .

File: search_frontend/js/search.js [hunk 2]
```diff
@@ -30,20 +31,50 @@
...
       this._total = ko.observable(0);
       this._activateTab = ko.observable(false);
     }
-
-    /** @todo fetch from api */
+        /* @todo find what type of node is the current tab*/
+    findStructure() {
+      var list = [] 
+        for (var key in mockup_data.filterData) {
+          if (key == this._node_type ) {
```
Comment (junior1q94) [inline search_frontend/js/search.js]: Unnecessary loop, you could have done this:
```
if (mockup_data.filterData.hasOwnProperty(this._node_type)) {
  // Do stuff with  mockup_data.filterData[this._node_type]
}
```

File: search_frontend/js/search.js [hunk 3]
```diff
@@ -30,20 +31,50 @@
...
+      this._nodeSearchData = ko.observableArray();
+      this.structureData = ko.observableArray()
       this._page = ko.observable(0);
       this._total = ko.observable(0);
       this._activateTab = ko.observable(false);
     }
-
-    /** @todo fetch from api */
+        /* @todo find what type of node is the current tab*/
+    findStructure() {
```
Comment (junior1q94) [inline search_frontend/js/search.js]: It seems what is being done here only need to be done once. The constructor is a good place for this.

File: search_frontend/index.html [hunk 4]
```diff
@@ -101,11 +101,8 @@ <h1 class="display-3 mb-5">Paradise Papers Search</h1>
                   </tr>
                 </thead>
                 <tbody data-bind="foreach: _nodeSearchData">
-                  <tr>
-                    <th scope="row">1</th>
-                    <td>Mark</td>
-                    <td>Otto</td>
-                    <td>@mdo</td>
+                  <tr data-bind="foreach: $data">
+                    <td data-bind="text: $data" ></td>
```
Comment (oscarpolanco) [inline search_frontend/index.html]: Extra space here

File: search_frontend/js/search.js [hunk 5]
```diff
@@ -30,20 +31,50 @@
...
+      }
+        /** @todo fetch from api */
     fetch () {
-      let data = mockup_data[this._node_type].data;
-      data.forEach(row => {
-        this._nodeSearchData.push(row);
-      });
-      this._page(this._page()+1);
-    }
+          this.findStructure(mockup_data.filterData);
```
Comment (junior1q94) [inline search_frontend/js/search.js]: Each time we call fetch, findStructure is executed. This should be done in the constructor.

**Review by junior1q94 — CHANGES_REQUESTED** [review]
Great work making the table rows dynamic! We just need to refactor now. @mariamera

File: search_frontend/index.html [hunk 6]
```diff
@@ -93,19 +93,15 @@ <h1 class="display-3 mb-5">Paradise Papers Search</h1>
...
                 </thead>
+                <!-- For each row in _nodeSearchData add the node property related data-->
                 <tbody data-bind="foreach: _nodeSearchData">
-                  <tr>
-                    <th scope="row">1</th>
-                    <td>Mark</td>
-                    <td>Otto</td>
-                    <td>@mdo</td>
+                  <tr data-bind="foreach: $parent._nodePropertyList">
+                    <td data-bind="text: $parent[$data.property_name]" >sd</td>
```
Comment (oscarpolanco) [inline search_frontend/index.html]: Extra space here

**Review by oscarpolanco — APPROVED** [review]
LGTM

**Review by junior1q94 — APPROVED** [review]

**Pull request merged**